src/services/listCourtCases.ts

```diff
@@ -14,11 +13,13 @@ const listCourtCases = async (
   const pageNumValidated = (pageNum ? parseInt(pageNum, 10) : 1) - 1 // -1 because the db index starts at 0
   const maxPageItemsValidated = maxPageItems ? parseInt(maxPageItems, 10) : 25
   const courtCaseRepository = connection.getRepository(CourtCase)
+  const orderByQuery = `courtCase.${orderBy ?? "errorId"}`
   const query = courtCasesByVisibleForcesQuery(courtCaseRepository, forces)
     .leftJoinAndSelect("courtCase.triggers", "trigger")
-    .orderBy(`courtCase.${getColumnName(courtCaseRepository, orderBy ?? "errorId")}`, order === "desc" ? "DESC" : "ASC")
-    .offset(pageNumValidated * maxPageItemsValidated)
-    .limit(maxPageItemsValidated)
+    .leftJoinAndSelect("courtCase.notes", "note")
+    .orderBy(orderByQuery, order === "desc" ? "DESC" : "ASC")
```

Comment:
I wonder if it actually works for the fields that have a different name from the actual database column name.